### DISCUSSION TITLE
Contribution graph wants to show contributions from the future for visitors who is not logged in

### DISCUSSION THREAD

**Discussion by abirana** [community/community#4331] [Other Feature Feedback, Questions, & Ideas]
I found one issue when viewing contribution graph of any user, for visitors who is not logged in.

Reproduce this with following steps:
1. First you should be logged out of Github or just view using incognito mode
2. Go to someone's profile or just your own, I went to mine https://github.com/abirana
3. Below the **contribution graph**, I've 2 team/project that I've contributed to (You might have different)
4. Click on one of the team, it should show contribution to that team. And above the **contribution graph** you should see **## contribution in 2021**
Note: It is fine until here but keep an eye on the year
6. Now to deselect the team, click on selected team name again
7. Now check the title you should see **0 contributions in 2022**. If you select a year from the side, then it can reach up to 2023.

I've recorded the event so you can check it here as well.
https://user-images.githubusercontent.com/20278933/123474588-e3d75a80-d619-11eb-9d9a-a1a0849adb86.mp4

---
Note: I've already reported the issue through the support few weeks ago and I got reply that it has been passed on to the engineers. I saw that it has still not been solved and today I found about [Isaacs GH Repo](https://github.com/isaacs/github) so I posted there which seemed to be the best place to keep it in public. Then again within that repo I found about this new official Github's Feedback repo, so I'm posting this here again. 

Let me know, if it is best to remove the issue from [Isaacs GH Repo](https://github.com/isaacs/github)

**Comment by alimttth** [comment]
as for unlimited links, you can use the default README on your landing page to include all of your links, you can even make it beautiful by making a link icon instead of the actual link  this is what it looks like
